peps/pep-0733.rst

```diff
@@ -0,0 +1,628 @@
+PEP: 733
+Title: An Evaluation of Python's Public C API
+Author: Erlend E. Aasland <erlend@python.org>, Domenico Andreoli <domenico.andreoli@linux.com>, Stefan Behnel <stefan_ml@behnel.de>, Carl Friedrich Bolz-Tereick <cfbolz@gmx.de>, Simon Cross <hodgestar@gmail.com>, Steve Dower <steve.dower@python.org>, Tim Felgentreff <tim.felgentreff@oracle.com>, David Hewitt <1939362+davidhewitt@users.noreply.github.com>, Shantanu Jain <hauntsaninja at gmail.com>, Wenzel Jakob <wenzel.jakob@epfl.ch>, Irit Katriel <irit@python.org>, Marc-Andre Lemburg <mal@lemburg.com>, Donghee Na <donghee.na@python.org>, Karl Nelson <nelson85@llnl.gov>, Ronald Oussoren <ronaldoussoren@mac.com>, Antoine Pitrou <solipsis@pitrou.net>, Neil Schemenauer <nas@arctrix.com>, Mark Shannon <mark@hotpy.org>, Stepan Sindelar <stepan.sindelar@oracle.com>, Gregory P. Smith <greg@krypto.org>, Eric Snow <ericsnowcurrently@gmail.com>, Victor Stinner <vstinner@python.org>, Serhiy Storchaka <storchaka@gmail.com>, Petr Viktorin <encukou@gmail.com>, Carol Willing <willingc@gmail.com>, William Woodruff <william@yossarian.net>, David Woods <dw-git@d-woods.co.uk>, Jelle Zijlstra <jelle.zijlstra@gmail.com>
```

Comment:
As others have noticed, this is a _really_ long list of authors for a PEP, much longer than any I can recall before, which seems potentially quite difficult to read and manage, since you'll need to get approval from all of them to post the PEP, and if changes are proposed, how do you ensure everyone agrees on them? However, there isn't any hard limit on author count, so up to you if you want to manage that.

To make that a _bit_ easier, I've at least formatted the header properly. You might also want to sort it lexicographically by given name, surname or equivalent to make it easier to search for and find a specific person; you're welcome to edit my comment if so.

```suggestion
Author: Erlend E. Aasland <erlend@python.org>,
        Domenico Andreoli <domenico.andreoli@linux.com>,
        Stefan Behnel <stefan_ml@behnel.de>,
        Carl Friedrich Bolz-Tereick <cfbolz@gmx.de>,
        Simon Cross <hodgestar@gmail.com>,
        Steve Dower <steve.dower@python.org>,
        Tim Felgentreff <tim.felgentreff@oracle.com>,
        David Hewitt <1939362+davidhewitt@users.noreply.github.com>,
        Shantanu Jain <hauntsaninja at gmail.com>,
        Wenzel Jakob <wenzel.jakob@epfl.ch>,
        Irit Katriel <irit@python.org>,
        Marc-Andre Lemburg <mal@lemburg.com>,
        Donghee Na <donghee.na@python.org>,
        Karl Nelson <nelson85@llnl.gov>,
        Ronald Oussoren <ronaldoussoren@mac.com>,
        Antoine Pitrou <solipsis@pitrou.net>,
        Neil Schemenauer <nas@arctrix.com>,
        Mark Shannon <mark@hotpy.org>,
        Stepan Sindelar <stepan.sindelar@oracle.com>,
        Gregory P. Smith <greg@krypto.org>,
        Eric Snow <ericsnowcurrently@gmail.com>,
        Victor Stinner <vstinner@python.org>,
        Serhiy Storchaka <storchaka@gmail.com>,
        Petr Viktorin <encukou@gmail.com>,
        Carol Willing <willingc@gmail.com>,
        William Woodruff <william@yossarian.net>,
        David Woods <dw-git@d-woods.co.uk>,
        Jelle Zijlstra <jelle.zijlstra@gmail.com>
```

Reply:
Just for fun, using the PEP "API":

```python
import pandas as pd
import requests

peps = requests.get("https://peps.python.org/api/peps.json").json()
df = pd.DataFrame.from_dict(peps, orient="index")
```

I can see the longest existing authors field is 197 characters (as a comm-separated list of just the names):

```pycon
>>> max(df["authors"].str.len())
197
```

This PEP's authors has a length a bit over twice that:

```pycon
>>> len("Erlend E. Aasland, Domenico Andreoli, Stefan Behnel, Carl Friedrich Bolz-Tereick, Simon Cross, Steve Dower, Tim Felgentreff, David Hewitt, Shantanu Jain, Wenzel Jakob, Irit Katriel, Marc-Andre Lemburg, Donghee Na, Karl Nelson, Ronald Oussoren, Antoine Pitrou, Neil Schemenauer, Mark Shannon, Stepan Sindelar, Gregory P. Smith, Eric Snow, Victor Stinner, Serhiy Storchaka, Petr Viktorin, Carol Willing, William Woodruff, David Woods, Jelle Zijlstra")
447
```

Reply:
Your suggestion does make it a lot more readable, thanks. Note that it's already sorted by last name.